TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/datahandlers/NettyPacketToHttpConsumerTest.java

```diff
@@ -103,10 +109,12 @@ public void testThatTestSetupIsCorrect()
     {
         for (var useTls : new boolean[]{false, true}) {
             try (var client = new SimpleHttpClientForTesting(useTls)) {
-                var endpoint = testServers.get(useTls).localhostEndpoint();
-                var response = makeTestRequestViaClient(client, endpoint);
-                Assertions.assertEquals(SERVER_RESPONSE_BODY,
+                try (var testServer = createTestServer(useTls, false)) {
```

Comment:
nit: you can put multiple try-with-resources within the same try by separating them with a semicolon...
```
            try (var client = new SimpleHttpClientForTesting(useTls);
                   var testServer = createTestServer(useTls, false)) {
```

Reply:
Done, thanks